src/Backend/Modules/MediaLibrary/Domain/MediaItem/MediaItem.php

```diff
@@ -218,6 +223,36 @@ private function setResolutionFromPath(string $path)
         }
     }
 
+    private function setResolutionForSvg(string $path)
+    {
+        if ($this->getType()->isImage()) {
```

Comment:
Can you rewrite it to 
`if (!$this->getType()->isImage()) {return;}`
That will create less indentation and makes it easier to see what happens, or maybe even throw a LogicException since you shouldn't be calling that method for non images

Reply:
See https://github.com/forkcms/forkcms/pull/3424/commits/92d1accc3a53edab9602b64cde92d4e13513a505